FrameworkSystem/Client/ProxyManagerClient.py

```diff
@@ -125,12 +157,17 @@ def setPersistency(self, userDN, userGroup, persistent):
                             record)
     return retVal
 
-  def uploadProxy(self, proxy=False, diracGroup=False, chainToConnect=False, restrictLifeTime=0, rfcIfPossible=False):
-    """
-    Upload a proxy to the proxy management service using delegation
+  def uploadProxy(self, proxy=False, restrictLifeTime=0, rfcIfPossible=False):
```

Comment:
Why removing the `diracGroup` from the list of possible parameters ? Isn't that a useful option ? 
Also, changing this interface should be very advertised to the extensions. You may want to do it in a bit more backward compatible way (like keep the parameters, but throw a warning if its set)

Reply:
In this PR all proxies upload without dirac group extension.

Reply:
I am not sure I understand. If I am not mistaken, you are adding the possibility to upload proxies without DIRAC group. That's fine. But you also seem to remove the possibility to upload proxy WITH DIRAC group. Is that correct or do I misunderstand ?

Reply:
Its correct